CefSharp.MinimalExample.Wpf/CefSharp.MinimalExample.Wpf.csproj

```diff
@@ -67,8 +67,10 @@
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Expression.Interactions, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
```

Comment:
They are set through "Add Reference...", not something that I download and add as a Reference, but still not something that comes right out of the box.

I can make an Attached Property Behavior instead if you prefer, it should work just out of the box, just not something Blend could interact with nicely.